src/url-params.ts

```diff
@@ -99,7 +129,14 @@ export class AnalyticsUrlParams {
 	}
 
 	/**
-	 * @return part of a query parameter string that can be appended to URLs
+	 * @return session ID that can be passed to other sites
+	 */
+	getSessionId() {
+		return this.sessionId;
```

Comment:
```suggestion
		return this.client ? this.client.sessionId() : this.sessionId;
```

It's more reliable to depend on Amplitude as the single source of truth. Besides, `this.sessionId` would be undefined / null if there is no incoming `sessionId`.

Reply:
Sounds good.